tests/acceptance/features/bootstrap/PublicWebDavContext.php

```diff
@@ -230,29 +232,47 @@ public function thePublicRenamesFileFromTheLastPublicShareUsingThePasswordPasswo
 	 * @return void
 	 */
 	public function downloadPublicFileInsideAFolder(string $path, string $publicWebDAVAPIVersion = "old"):void {
-		$this->publicDownloadsTheFileInsideThePublicSharedFolderWithPasswordAndRange(
+		$response = $this->downloadTheFileInsideThePublicSharedFolder(
 			$path,
 			"",
 			"",
 			$publicWebDAVAPIVersion
 		);
+		$this->featureContext->setResponse($response);
 	}
 
 	/**
-	 * @When /^the public downloads file "([^"]*)" from inside the last public link shared folder with password "([^"]*)" using the (old|new) public WebDAV API$/
+	 * @When /^the public downloads file "([^"]*)" from inside the last public link shared folder using the (old|new) public WebDAV API with password "([^"]*)"$/
 	 *
 	 * @param string $path
-	 * @param string|null $password
 	 * @param string $publicWebDAVAPIVersion
+	 * @param string $password
 	 *
 	 * @return void
 	 */
-	public function publicDownloadsTheFileInsideThePublicSharedFolderWithPassword(
+	public function downloadPublicFileInsideAFolderWithPassword(string $path, string $publicWebDAVAPIVersion = "old", string $password = ""):void {
+		$response = $this->downloadTheFileInsideThePublicSharedFolder(
+			$path,
+			$password,
+			"",
+			$publicWebDAVAPIVersion
+		);
+		$this->featureContext->setResponse($response);
+	}
+
+	/**
+	 * @param string $path
+	 * @param string|null $password
+	 * @param string $publicWebDAVAPIVersion
+	 *
+	 * @return ResponseInterface
+	 */
+	public function downloadTheFileInsideThePublicSharedFolderWithPassword(
```

Comment:
```suggestion
	public function downloadFileFromPublicFolderWithPassword(
```

Reply:
Could this be removed and replaced by `downloadTheFileInsideThePublicSharedFolder`?